src/common/main/kotlin/me/owdding/skyblockpv/utils/Utils.kt

```diff
@@ -188,4 +179,10 @@ object Utils {
     fun <T : Any> JsonElement?.toDataOrThrow(codec: MapCodec<T>): T = this.toDataOrThrow(codec.codec())
 
     fun Matrix3x2fStack.copy() = Matrix3x2f(this)
+
+    fun RenderPipeline.Builder.withShaderDefine(name: String, array: IntArray): RenderPipeline.Builder {
+        val accessor = this as RenderPipelineBuilderAccessor
+        accessor.`sbpv$define`(name, "int[](${array.joinToString(", ") { it.toString() }})")
+        return this
+    }
```

Comment:
Method name should probably be more descriptive?

Reply:
It’s the same name as vanillas defines